### PR TITLE
[feature-2] Created a new branch for new functionality addition

### PR DESCRIPTION
**Pull request by gauravkhanna3007**
Created the following features in this new branch

1. Added a new folder
2. Added a new file in this folder

### CODE REVIEW

File: new-folder-feature-2/new-file-feature2_2.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Second line created in feature-2 branch
```
Comment (YohaanK) [inline new-folder-feature-2/new-file-feature2_2.txt:1]: This new file addition was a masterpiece.

Comment (gauravkhanna3007) [reply new-folder-feature-2/new-file-feature2_2.txt:1]: thanks for your appreciation, much needed indeed.

File: new-folder-feature-2/new-file-feature2.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Created a new file in feature-2 branch
+
+Add another line in feature-2 branch
```
Comment (YohaanK) [inline new-folder-feature-2/new-file-feature2.txt:3]: Will this code work under the new platform? Have you tested it under the new platform? please confirm this

Comment (gauravkhanna3007) [reply new-folder-feature-2/new-file-feature2.txt:3]: yes it work and i have tested it thoroughly.

**Pull request merged**